python/ray/autoscaler/log_timer.py

```diff
@@ -0,0 +1,66 @@
+import datetime
```

Comment:
We're changing Ray logging overall with this PR: https://github.com/ray-project/ray/pull/3691. @ls-daniel can you leave some comments? It will largely interact with these logging changes.

For this file, I think `logTimer` is a good addition. Is there a particular reason why we should centralize the logging calls in one file rather than using `logger = logging.getLogger(__name__)` in each file?

Reply:
Hm yeah, I think most of the benefits added here are duplicated with that PR. Also one problem with moving the logger calls to one file is that the file name / line numbers in the format string won't be correct.

Shall we keep logTimer and use the normal logger for the rest?

Reply:
I don't have a preference, personally. The additional log functions can be converted back to using logging (I used it as a sort of intermediate step to ensure I hadn't missed any logging functions)

The line numbers I don't see as being too important because the logging calls themselves are specific enough that you can just grep for the line (and that way it forces a more descriptive log message). Though that could rot over time.